guides/common/modules/proc_customizing-the-registration-templates.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Customizing_the_Registration_Templates_{context}"]
+= Customizing the Registration Templates
+
+You can customize the registration process by editing the provisioning templates.
+Note that all default templates in {Project} are locked.
+If you want to customize the registration templates, you must clone the default templates and edit the clones.
```

Comment:
I am unsure about the wording here. IMHO we should recommend to _not_ change any templates because they'll not get updates in the future and are (probably?) not 100% supported by RH/ATIX.

Do we have a generic warning that we can include here?

Reply:
Good point. I'll probably create one.